src/SwedbankPay.Sdk/Resources/ConsumerResource.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace SwedbankPay.Sdk.Resources
+{
+    using System;
+    using System.Threading.Tasks;
+    using RestSharp;
+    using SwedbankPay.Sdk.Exceptions;
+    using SwedbankPay.Sdk.Models;
+    using SwedbankPay.Sdk.Models.Request;
+    using SwedbankPay.Sdk.Models.Response;
+
+    public class ConsumerResource : ResourceBase, IConsumerResource
```

Comment:
`ConsumerResource` belongs inside the `Consumers` namespace alongside everything else related to consumers.